library/src/main/java/com/karumi/headerrecyclerview/HeaderRecyclerViewAdapter.java

```diff
@@ -46,7 +46,7 @@
    * Invokes onCreateHeaderViewHolder, onCreateItemViewHolder or onCreateFooterViewHolder methods
    * based on the view type param.
    */
-  @Override public VH onCreateViewHolder(ViewGroup parent, int viewType) {
+  @Override public final VH onCreateViewHolder(ViewGroup parent, int viewType) {
```

Comment:
If they need it they should not use this library. This library offers one header and one footer, the rest of the features should be implemented using another more complex library like Renderers.